scavenger-agent-java/build.gradle.kts

```diff
@@ -72,6 +73,39 @@ dependencies {
     testImplementation("org.mockito:mockito-inline:4.3.1")
 }
 
+testSets {
+    create("integrationTest")
+}
+
+dependencies {
+    "integrationTestImplementation"("org.springframework.boot:spring-boot-starter-test:2.5.12")
```

Comment:
" " is necessary to use integrationTestImplementation?

Reply:
Yes, it is needed since the name `integrationTestImplementation` is dynamically created by `org.unbroken-dome.test-sets` gradle plugin.

Related: https://github.com/unbroken-dome/gradle-testsets-plugin/issues/109#issuecomment-694181706

Reply:
Yes, it is needed since the name integrationTestImplementation is dynamically created by org.unbroken-dome.test-sets gradle plugin.

[Related issue](https://www.github.com/unbroken-dome/gradle-testsets-plugin/issues/109#issuecomment-694181706)